_ext/bandcamp.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Mopidy-Bandcamp
+service: bandcamp
```

Comment:
This field isn't needed. It is only used by Spotify and SoundCloud that has custom authentication workflows running on mopidy.com.

```suggestion
```

Reply:
Thanks!